.gitattributes

```diff
@@ -1,4 +1,6 @@
 * text eol=lf
+*.eml eol=crlf
```

Comment:
This is needed because otherwise git would change `\r\n\` to `\n` as stated in the line above. Emails (as HTTP does) always use Windows `\r\n` CRLF always but they are text files.